ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -75,11 +74,11 @@ void RSkComponentImage::OnPaint(
    /*Draw Image Shadow*/
     if(contentShadow) {
         if(imageProps.resizeMode == ImageResizeMode::Repeat) {
-            sk_sp<SkImageFilter> imageFilter(SkImageFilters::Tile(targetRect,frameRect,layer()->shadowFilter));
```

Comment:
I assume , change in third argument from shadowFilter to nullptr, is because combining shadow filter & tile is not working as expected. If thta it, current code, will not apply any shadow for resize mode. 
So suggest to remove the codes and add TO DO